services/app-web/src/testHelpers/apolloMocks/contractPackageDataMock.ts

```diff
@@ -1866,6 +1866,370 @@ function mockContractPackageUnlocked(
     }
 }
 
+function mockContractPackageUnlockedWithUnlockedType(
```

Comment:
I'm aware this is very close to an existing mock. However switching out the existing mock to expect and return an unlockedContract type would have been a bit of a refactor since it's being used in multiple places. If folks feel like it is in scope to make that change, lmk and I can include it as a part of this PR

Reply:
Thanks for calling this out and paying attention to tests. My vote is let's land this work, ship,  but then you immediately follow on with that cleanup refactor PR during this sprint while its in mind.